.github/workflows/upgrade-e2e.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Upgrade Tests
+
+on:
+  pull_request:
+
+jobs:
+  upgrade-e2e:
+    name: Verify that a cluster upgraded from release continues working
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+      - run: git fetch origin +refs/tags/*:refs/tags/*
```

Comment:
We need this extra fetching logic?

Reply:
No, we don’t, I meant to remove it and forgot!